Features/Stories/Fieldtypes/AuthorsFieldType.feature

```diff
@@ -0,0 +1,96 @@
+Feature: Test the validations done on Author fieldtype from Editorial Interface - Authors fieldtype
+
+    @javascript
+    Scenario: Message when using empty author field
+        Given I am logged in as admin on PlatformUI
+        And a Content Type exists with identifier "Author" in Group with identifier "Content" with fields:
+            | identifier | Type     | Name   |
+            | name       | ezstring | Name   |
+            | author     | ezauthor | Author |
+        And I create a content of content type "Author" with:
+            | Name       |
+            | AuthorTest |
```

Comment:
to ease the usage of the test report, it would be nice to use more meaningful names, "AuthorTest" appears 5 times in this PR. Here, the name could be "Empty author test" for instance.